spec/lib/zendesk_apps_tools/api_connection_spec.rb

```diff
@@ -134,4 +136,33 @@ def initialize(subdomain = nil, username = nil)
       end
     end
   end
+
+  describe '#get_connection' do
+    context 'with missing credentials' do
+      it 'calls prepare_api_auth for password when password is missing' do
+        subject = subject_class.new('subdomain', 'username')
+        expect(subject).to receive(:prepare_api_auth).exactly(1).times
```

Comment:
Is it possible to narrow the scope of this test (and the one below) so that we're testing it is `received` with certain parameters?

Reply:
Hi @fvzzy , If you have sometime today, I might grab you in person to elaborate more on this?

Reply:
As discussed in real life, I will rename the two tests so that it reads better. 👍